dev/travis/script.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+# Copyright © 2015 Magento. All rights reserved.
+# See COPYING.txt for license details.
+
+set -e
+export PATH="./../../../vendor/bin:$PATH"
```

Comment:
I agree, nobody is forced to use PATH.
I'd prefer project's `vendor` vs. user's `bin`, as it guarantees that there is nothing else there, though `bin` might be in PATH by default, so I think it should be OK.